synopsys-task/task.json

```diff
@@ -477,6 +553,26 @@
       "visibleRule": "blackduckReportsSarifCreate = true",
       "groupName": "scanOptions"
     },
+    {
+      "name": "blackduckAzureToken",
+      "type": "string",
+      "label": "Azure Token",
+      "defaultValue": "",
+      "required": false,
+      "helpMarkDown": "PAT or System.AccessToken has to be passed with legitimate access when Coverity/BlackDuck/Polaris PR comment and BlackDuck FIX PR is enabled.",
```

Comment:
We should only mention about BlackDuck here since this section is BlackDuck specific but Polaris/Coverity is also mentioned.

Correct message should be - `PAT or System.AccessToken has to be passed with legitimate access when BlackDuck pull request comment or fix pull request is enabled.`